### DISCUSSION TITLE
There are two rectangles in a graphic instance.  Is there a way to directly delete or hide one of them?

### DISCUSSION THREAD

**Discussion by pcbren1987** [pixijs/pixijs#7779] [Q&A]
![2021-09-06_221839](https://user-images.githubusercontent.com/89827437/132232503-e1cd55de-4f6c-4759-a91b-c0cf46f6b839.png)

**Comment by domis86** [comment]
Yes - create second Graphics object:
```
const graphic2 = new PIXI.Graphics();
graphic2.beginFill(...
```
and add it to stage container similarily as first `graphic`

**Reply by pcbren1987** [reply]
There are 100k graphic graphics in the container, and it will be very slow to create an instance of each rectangle.

**Reply by domis86** [reply]
i see @eXponenta already answered to you in July : https://github.com/pixijs/pixijs/discussions/7577#discussioncomment-959560 - proposed to use "mesh". Please search for info about mesh in discussions or issues in this github project (example: https://github.com/pixijs/pixijs/issues/7755 )
